bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/AuthFilter.java

```diff
@@ -266,29 +268,33 @@ public void filter(@Nullable ContainerRequestContext requestContext) throws IOEx
                             }
                         }
                     }
-                } else if (isImplicitUserRole(requestContext)) {
+                } else if (isImplicitUserRole(servletRequest)) {
                     requestContext.setSecurityContext(new AnonymousUserSecurityContext());
                 }
             } catch (AuthenticationException e) {
-                logger.warn("Unauthorized API request from {}: {}", getClientIp(requestContext), e.getMessage());
+                logger.warn("Unauthorized API request from {}: {}", getClientIp(servletRequest), e.getMessage());
```

Comment:
Is there any particular reason why we do not log stack trace here?

Reply:
No, I think that maybe avoid emitting too much log in case of an attack? That makes sense for me.

Reply:
I tend to agree with @GiviMAD. Do you see any additional information from the stack trace that is lost otherwise?